modules/020-cni-cilium/images/cilium/Dockerfile

```diff
@@ -0,0 +1,76 @@
+# syntax=docker/dockerfile:1.2
+
+# Copyright 2020-2021 Authors of Cilium
+# SPDX-License-Identifier: Apache-2.0
+
+ARG CILIUM_BUILDER_IMAGE=quay.io/cilium/cilium-builder:9005878f5668f3f9bf49b237c9b8b37e248c33d6@sha256:682fc385c494984c2822ac1f2bc806e504addfdbd7a3257bfa587c0938b7614f
```

Comment:
`cilium-builder` is composed of some other images. It would be a royal PITA to collect all them and merge them in our build process. Do we absolutely _need_ to duplicate their build process?

https://github.com/cilium/cilium/blob/8cfd550ba836b4421784a0dbf427354f0a949054/images/builder/Dockerfile

Reply:
Done.